INSTALL.md

```diff
@@ -0,0 +1,72 @@
+# Install prerequisites
+
+The following works for Debian Buster. Steps would be extremely close
+on recent Ubuntus but exact package names may vary and will
+definitely vary on distros that aren’t Debian derivatives.
+
+```
+$ sudo apt-get update && apt-get upgrade -y && apt-get install -y \
+    libcanberra-gtk3-module dbus-x11 ninja-build virtualenv \
+    libxml2-dev pkg-config libglib2.0-dev libgoocanvas-2.0-dev \
+    libwebkit2gtk-4.0-dev libgtksourceview-3.0-dev libgspell-1-dev \
+    libplist-dev desktop-file-utils libgstreamer1.0-dev \
+    gstreamer1.0-plugins-bad gstreamer1.0-plugins-good \
+    gstreamer1.0-tools git build-essential
+```
+## Meson
+
+gnome-inform7's requires meson >= 0.55.0. Debian Buster's is too old,
+but if you're or Debian Bullseye (or Sid) or Ubuntu 20.10 (or 21.04)
+could just apt install meson.
+
+```
+$ git clone -b gtk3 https://github.com/ptomato/gnome-inform7.git
+$ cd gnome-inform7
+$ virtualenv --python=python3 meson
+$ source meson/bin/activate
+$ pip install meson==0.55
+```
+At this point you have to manually copy the appropriate ni binary
+for your architecture to gnome-inform7/src/ni/ni -– if you already
+have Inform 7 installed, you already have a copy, probably in
+/usr/local/libexec/ni , otherwise find it in
+
+http://inform7.com/apps/6M62/I7_6M62_Linux_all.tar.gz
+
+Once that’s done (still in the same gnome-inform7 directory as above):
+
+```
+$ meson setup build
+$ cd build
+$ meson compile
+$ sudo meson install # WARNING! will overwrite any existing gnome-inform7 under /usr/local
+
+# now you can run it with
+$ LD_LIBRARY_PATH=/usr/local/lib/x86_64-linux-gnu gnome-inform7
+# or if you don't do meson install, run with:
```

Comment:
I did put in a script that should do this for you if you want to run it uninstalled:
```
$ ./build-aux/run_uninstalled.sh
```
(Basically the same as this `gi7` script, only for uninstalled running...)

Another thing you could do is install it into `~/.local/bin`, I believe this location is in the PATH on any new-ish Linux distribution, doesn't require root privileges to install, and shouldn't require setting LD_LIBRARY_PATH. (I haven't checked that though.)
```
$ meson setup build -Dprefix=$HOME/.local
$ cd build
$ meson compile
$ meson install
$ gnome-inform7
```